### PR TITLE
[icalendar] Provide example for Nextcloud usage

### PR DESCRIPTION
**Pull request by t2000**
Users of the icalender OH 1 binding might wonder how to get the url for the
ical file for their calenders. It took me a couple of hours to find this, so
I think it might be useful to have it as an example in the documentation.

Signed-off-by: Stefan Triller <github@stefantriller.de>

### CODE REVIEW

File: bundles/org.openhab.binding.icalendar/README.md [hunk 1]
```diff
@@ -166,6 +166,11 @@ BEGIN:Calendar_Test_Switch:ON
 END:Calendar_Test_Switch:OFF
 ```
 
+### Notes for nextcloud
```
Comment (kaikreuzer) [inline bundles/org.openhab.binding.icalendar/README.md:169]:
```suggestion
### Notes for Nextcloud
```

**Review by kaikreuzer — APPROVED** [review]
Thanks! As this is a pure documentation update, there is no risk merging it for 3.0.

**Comment by daMihe** [comment]
The documented text is wrong in case an external authentication is used (which is usual in greater installations). My question is: is the OH-Addon-Documentation the right place for documenting 3rd-party software? If yes, i'll add a tutorial for getting the right URL here - but it does not feel right as Nextcloud may change independently.

Shouldn't be this a wiki inside the forum?

Sorry i did not get fast enough access to an instance to verify this behaviour (that's why i did not stop the PR before).

**Comment by t2000** [comment]
@daMihe thanks for your comment. Indeed I was wondering whether the Nextcloud information should be part of the example here.

However, I think that this icalendar binding is really new and since it's a replacement for the OH-1 caldav bindings that have been removed from OH3, a lot of openHAB users might wonder how to migrate. So maybe the example I gave here belongs to the migration guide from OH1-caldav -> OH3-icalendar binding, if there is such a migration guide.

The reason why I think that this is important is simply the fact that the OH1 caldav binding did not need the `export` parameter set and in the new binding this is required!

And for home users that use the "normal" Nextcloud authorization, the new OH3-icalendar binding works as I have described it (I have tested it prior to creating this documentation PR).

**Comment by daMihe** [comment]
@t2000 Okay. There is currently no guide as icalendar was never intended to be a replacement (i just needed the feature "presence switch"), it also has less features as implementing them is simply not possible without DAV-access (think about the recording feature). However, as there is no current caldav replacement, so maybe a replacement guide is a good idea. But i think the forum is a better place for it. Do have a bit time for talking an hour or two next days? I think discussing what should be in such a post and writing at least some bullet points it is easier live.

**Comment by t2000** [comment]
@daMihe Feel free to write a community post for a migration.

I do not use any "DAV" functionality of the caldav binding, other than reading dates for appointments from the calendar. So the only thing that I can contribute is what I have written in this PR, i.e. "add the `export` to the URL.

If you use more features of the OH-1 caldav binding, I would appreciate if you could write a more comprehensive post. But as I said I won't be able to contribute much :) However, I can read it and comment if you would like that.

**Comment by daMihe** [comment]
Done [there](https://community.openhab.org/t/small-guide-to-icalendar/111430?u=damihe).

@kaikreuzer Would you revert this PR? I leave this up to you - in my eyes addon docs are just the wrong place for documenting Nextcloud. And now an extended text exists.

**Comment by t2000** [comment]
@daMihe Thank you very much for the comprehensive documentation on the community site!

**Pull request merged**